osquery/logger/plugins/windows_event_log_manifest/osquery.man

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<instrumentationManifest xsi:schemaLocation="http://schemas.microsoft.com/win/2004/08/events eventman.xsd" xmlns="http://schemas.microsoft.com/win/2004/08/events" xmlns:win="http://manifests.microsoft.com/win/2004/08/windows/events" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:trace="http://schemas.microsoft.com/win/2004/08/events/trace">
+	<instrumentation>
```

Comment:
nit: Does this file require having the wide tab character spacing? Can we change the file to use 2 character spaces or is this auto-generated?

Reply:
This file is automatically generated by a tool called Manifest Generator (ecmangen.exe). You can find it inside the Windows SDK folder.